packages/table/components/HeaderCell.tsx

```diff
@@ -12,20 +12,11 @@ export interface HeaderCellProps extends CellProps {
   lowerCase?: boolean;
 }
 
-export class HeaderCell extends React.PureComponent<HeaderCellProps, {}> {
-  public render() {
-    const HeaderCell = this.props.lowerCase
-      ? styled(Cell)`
-          ${headerCellCss};
-          ${textTruncate};
-        `
-      : styled(Cell)`
-          ${headerCellCss};
-          ${textCapitalize};
-          ${textTruncate};
-        `;
-    return <HeaderCell>{this.props.children}</HeaderCell>;
-  }
-}
+const HeaderCell = (props: HeaderCellProps) => (
+  <Cell lowerCase={props.lowerCase} {...props} />
```

Comment:
Why do we have to specifically pass `lowerCase`? Does it not come through with `...props`?

Reply:
I did not think of that, it probably does.